recipes/open62541/all/conanfile.py

```diff
@@ -181,7 +182,7 @@ def requirements(self):
         if self.options.encryption == "mbedtls":
             self.requires("mbedtls/2.25.0")
         elif self.options.encryption == "openssl":
-            self.requires("openssl/1.1.1s")
+            self.requires("openssl/[>=1.1 <4]")
```

Comment:
CCI now allows version ranges for a few recipes, namely openssl and cmake, so let's use it. I dont think bumping the version to openssl 3 here hurts

Also, I did not bump the other requirements to their latest available versions because I ran out of time for today, will do so on a next pass